README.md

```diff
@@ -574,6 +611,59 @@ expect(myComponent).toContainHtml('<span>Hello world!</span>');
 
 ## FAQ
 
+### Can I use this with a setup that aliases preact to react?
+
+The complete API compatibility with `@shopify/react-testing` means you can do the same kind of module aliasing you can do with `react` and `preact` with `@shopify/react-testing` and `@shopify/preact-testing`. If you are using jest, you would:
+
+1. Install `@shopify/preact-testing`
+
+```
+$ yarn add @shopify/preact-testing
+```
+
+2. Add a `moduleNameMapper` object to your `jest.config.js` config with the following keys
+
+```jsx
+moduleNameMapper: {
+  '^react$': 'preact/compat',
+  '^@shopify/react-testing$': '@shopify/preact-testing',
+  '^react-dom$': 'preact/compat',
+  '^react-dom/test-utils$': 'preact/test-utils',
+}
+```
+
+3. Create a `setupFilesAfterEnv` entry and setup the testing library there
+
+```tsx
+import 'preact/debug';
+import '@shopify/preact-testing/matchers';
+import {destroyAll} from '@shopify/preact-testing';
+
+// eslint-disable-next-line jest/require-top-level-describe
+afterEach(() => {
+  destroyAll();
+});
+```
+
+4. Import and use the library in your tests via the `@shopify/react-testing` alias you set up above
+
+```tsx
+import React from 'react';
+import {mount} from '@shopify/react-testing';
+import faker from 'faker';
+
+import {MyComponent} from '../MyComponent';
+import {UnstyledMyComponent} from '../components';
+
+describe('<MyComponent />', () => {
+  it('renders hello world', () => {
+    const link = mount(<MyComponent />);
+
+    expect(link).toContainReactText('hello world');
+  });
+});
+```
+
```

Comment:
this also feel like more of a general setup that can be omit or should be move to a general place instead of under this question.

Reply:
I think this one is ok to stay here because you would *not* import from `React` or use `toContainReactText` if you weren't doing the alias stuff